api-report/merge-tree.api.md

```diff
@@ -126,7 +126,7 @@ export class Client extends TypedEventEmitter<IClientEvents> {
     // (undocumented)
     getCollabWindow(): CollaborationWindow;
     // (undocumented)
-    getContainingSegment<T extends ISegment>(pos: number, op?: ISequencedDocumentMessage, localSeq?: number): {
+    getContainingSegment<T extends ISegment>(pos: number, op?: ISequencedDocumentMessage, localSeq?: number, seqNumberFrom?: number, clientId?: number): {
```

Comment:
this doesn't look right--rather than add `seqNumberFrom` and `clientId`, you should adjust the `op` argument to only require the portions of `op` that are actually used (i.e. `referenceSequenceNumber` and `clientId`) and plumb things through using those fields in sequence.